Create tasks for VS Code

MrHatter89 opened the pull request:
Added a task for VS Code to copy the results of the completed translation work directly to the game folder.
In file _.vscode\task.json_ you need to replace the path to your location of folder _Krumpac_Reforge_Translations_
Example:
` "command": "cp -r -Force ./plugins/* 'D:/SteamLibrary/steamapps/common/Valheim/BepInEx/plugins/Krumpac-Krumpac_Reforge_Translations'"`

MrHatter89 committed:
Create tasks.json

FelixReuthlinger reviewed (changes requested):
this is very unlikely to work for everyone, since you are referring an absolute path, even using "D:" drive, where most people will have it installed in C drive (also me). if you want to add some script like this, would require to use other variables to get the correct location.

MrHatter89 committed:
Update tasks.json

Finding the path with the game's Steam version in the registry.
Substituting the found path into the copy command.
Will not work with mod managers that create profiles outside the game directory.

MrHatter89 commented:
Finding the path with the game's Steam version in the registry.
Substituting the found path into the copy command.
Will not work with mod managers that create profiles outside the game directory.
